test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -811,16 +832,22 @@ public void testLogEntrySyncValidCrossTablesWithWritingAtSrc() throws Exception
         testConfig.clear();
         testConfig.setWritingSrc(true);
         testConfig.setDeleteOP(true);
-
         testConfig.setWaitOn(WAIT.ON_ACK);
-        startLogEntrySync(crossTables, WAIT.ON_ACK, false);
+
+        HashSet<WAIT> waitHashSet = new HashSet<>();
+        waitHashSet.add(WAIT.ON_ACK);
+        startLogEntrySync(crossTables, waitHashSet, true);
```

Comment:
why did this change? there is a method that takes only one condition.

Reply:
I want to put all things clear for each test case instead of using the wrapper, otherwise, while debugging, I need to jump around to see what are the real arguments passed.